File: app/src/main/java/dev/ragnarok/fenrir/db/model/entity/CityEntity.java
package dev.ragnarok.fenrir.db.model.entity;

import androidx.annotation.Keep;

@Keep
public class CityEntity {

    private int id;

    private String title;

    private boolean important;

    private String area;

    private String region;

    public int getId() {
        return id;
    }

    public CityEntity setId(int id) {
        this.id = id;
        return this;
    }

    public boolean isImportant() {
        return important;
    }

    public CityEntity setImportant(boolean important) {
        this.important = important;
        return this;
    }

    public String getArea() {
        return area;
    }

    public CityEntity setArea(String area) {
        this.area = area;
        return this;
    }

    public String getRegion() {
        return region;
    }

    public CityEntity setRegion(String region) {
        this.region = region;
        return this;
    }

    public String getTitle() {
        return title;
    }

    public CityEntity setTitle(String title) {
        this.title = title;
        return this;
    }
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/db/model/entity/CityEntity.kt
package dev.ragnarok.fenrir.db.model.entity

import androidx.annotation.Keep
import kotlinx.serialization.Serializable

@Keep
@Serializable
class CityEntity {
    var id = 0
        private set
    var title: String? = null
        private set
    var isImportant = false
        private set
    var area: String? = null
        private set
    var region: String? = null
        private set

    fun setId(id: Int): CityEntity {
        this.id = id
        return this
    }

    fun setImportant(important: Boolean): CityEntity {
        isImportant = important
        return this
    }

    fun setArea(area: String?): CityEntity {
        this.area = area
        return this
    }

    fun setRegion(region: String?): CityEntity {
        this.region = region
        return this
    }

    fun setTitle(title: String?): CityEntity {
        this.title = title
        return this
    }
}